models/service.go

```diff
@@ -73,7 +73,6 @@ type ServiceDefinitionList struct {
 }
 
 type ServiceDetails struct {
-	Cluster            string                                   `json:"cluster"`
```

Comment:
I'm not really sure why we embed the whole `Namespace` object in the `Service` model since it looks like just the name was being parsed anyways and you'd lose all the rest of the fields if you relied on e.g. `svc.Namespace.Labels`. Can we change `Namespace` to be a string and move this `Cluster` field to the `Service` model so it's more consistent with app/workload?

Reply:
The `Namespace` object was already there, and was the original source of confusion as I had assumed it would minimally have the cluster and namespace names set.  But the entire object had only the namespace name.  I left it in place because I was unsure if some code path may make use of the additional fields, but I can make the suggested change.  It's functionally equivalent, and if its more clear and consistent then we should do it.